keyboards/redragon/k630/keymaps/via/config.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright (C) 2023 Benjamín Ausensi
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Enable VIA RGB matrix and configure EEPROM emulation for VIA */
+#define VIA_QMK_RGBLIGHT_ENABLE
+#define WEAR_LEVELING_SN32_EMULATION_PAGE_COUNT 30
```

Comment:
```suggestion
```
this only needs to be overwritten from the default value, if the default value is deemed too small when building VIA. That is only the case on big keyboards, please verify if you need that